src/private/inventory.ts

```diff
@@ -103,6 +109,8 @@ export const inventoryPrivateHandlers: PrivateNatsHandler[] = [
                 ) as UpdateInventoryItemRequest;
 
                 try {
+                    await InventorySchema.validate({ inventory_item_id });
+                    await DataSchema.validate({ data });
```

Comment:
can you check if this is correct? When I try to update an inventory item with this request:

```
curl --location --request PUT 'http://localhost:8000/api/item-store/inventory/JnVuOnkB-8XRad6S5ee9' \
--header 'Content-Type: application/json' \
--data-raw '{
    "CLUB": "Racing Club Strasbourg"
}'
```

I get this error:

```
{
    "message": "The data (object of string(s)) must be provided."
}
```

although the payload is correct. I think this should be `await DataSchema.validate(data)` but not sure